cocotb/binary.py

```diff
@@ -295,22 +319,34 @@ def get_value_signed(self):
         else:
             return -1 * (1 + (int(~ival) & (signbit - 1)))
 
-    def set_value(self, integer):
-        self._str = self._convert_to[self.binaryRepresentation](integer)
+    get_value_signed = signed_integer.fget
 
-    @property
-    def is_resolvable(self):
-        """Does the value contain any ``X``'s?  Inquiring minds want to know."""
-        return not any(char in self._str for char in BinaryValue._resolve_to_error)
+    @signed_integer.setter
+    def signed_integer(self, val):
+        self.integer = val
 
-    value = property(get_value, set_value, None,
-                     "Integer access to the value. **deprecated**")
-    integer = property(get_value, set_value, None,
-                       "The integer representation of the underlying vector.")
-    signed_integer = property(get_value_signed, set_value, None,
-                              "The signed integer representation of the underlying vector.")
+    def get_hex_buff(self):
+        bstr = self.buff
+        hstr = '%0*X' % ((len(bstr) + 3) // 4, int(bstr, 2))
+        return hstr
+
+    def _adjust(self):
```

Comment:
Can you try to keep the code in the same order as before to keep the diff small? This function for instance did not change, but you've moved so much code from above it to below it that git thinks it did.